incubator/mattermost-team-edition/README.md

```diff
@@ -0,0 +1,76 @@
+# Mattermost Team Edition
+
+[Mattermost](https://mattermost.com/) is a hybrid cloud enterprise messaging workspace that brings your messaging and tools together to get more done, faster.
+
+## TL;DR;
+
+```bash
+$ helm install incubator/mattermost-team-edition
+```
+
+## Introduction
+
+This chart creates a [Mattermost](https://mattermost.com/) deployment on a [Kubernetes](http://kubernetes.io)
```

Comment:
Might be a good idea to stress that it's team edition so 👍 

>This chart creates a Mattermost deployment...

to

>This chart creates a Mattermost Team Edition deployment...

Reply:
done